spec/factories/documents.rb

```diff
@@ -3,13 +3,29 @@
     chunks { [] }
     user { association(:user) }
     collection { association(:collection) }
-    filename { "gnu_manifesto.md" }
     state { :chunked }
     vector_id { nil } # TODO: we probably don't need this
     chunking_profile { association(:chunking_profile) }
 
+    filename { "gnu_manifesto.md" }
+
     trait :with_file do
       file { Rails.root.join("spec/fixtures/files/small_doc.md") }
     end
   end
+  factory :document_from_web, class: 'Document' do
+    chunks { [] }
+    user { association(:user) }
+    collection { association(:collection) }
+    state { :chunked }
```

Comment:
Not a big deal at all, but if this is in state `chunked`, it would normally have chunks. What if the factory assumed an empty set of chunks and state `created`?